lib/stylesheet/manager/builder.rb

```diff
@@ -86,19 +90,19 @@ def source_map_fullpath
   end
 
   def source_map_filename
-    "#{stylesheet_filename}.map"
+    "#{stylesheet_filename}.map?__ws=#{current_hostname}"
```

Comment:
One curiosity: shouldn't `source_map_filename` just be `stylesheet_filename.sub(".scss", ".map")` or something like that? Why doesn't `stylesheet_filename` contain `__ws` too?

Reply:
Good point, this is quite confusing. I've just pushed a commit with some updated method names - does that look any better?

Reply:
This is better, but I still find it a bit strange that we assign a relative URL to a filename. I don't know what "right" would look like, so I think you can leave it this way.